roles/openshift_metrics/tasks/generate_certificates.yaml

```diff
@@ -118,13 +118,20 @@
   shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15
   register: hawkular_metrics_password
 - name: generate password for hawkular metrics jgroups
-  shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15 | base64
+  shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15
```

Comment:
this is to generate a random 15 characters, we wouldn't want to actually cat /dev/urandom with that filter...